tips-and-tricks/uv_install.md

```diff
@@ -0,0 +1,60 @@
+# UV Install on HPC
+
+UV is a new Python package manager which offers a few helpful features.
+[Website is here](https://docs.astral.sh/uv/)
+
+This short guide doesn't explain how to use UV generally but just how to get UV to work out of a directory, which is what you would want if you wanted the Python version stored in the project space.
+
+## Installation
+
+To install UV, run the following commands.
+It should be done in the project directory if you want all the users on the project to access it.
+
+`cd /go/to/project/directory`
+
+```bash
+mkdir uv_install
```

Comment:
Is there a reason to separate the `cd` out from the rest of the commands here? If not, I'd suggest we pull it inside the code block:

````suggestion
```bash
cd /go/to/project/directory

mkdir uv_install
````